test/unit/presenters/document_export_presenter_test.rb

```diff
@@ -49,9 +49,32 @@ class DocumentExportPresenterTest < ActiveSupport::TestCase
     assert_equal image.image_data.file_url, images.first['url']
   end
 
+  test "appends detailed document type information about an edition" do
```

Comment:
This should probably be rephrased to reflect that it's sub type rather than document type information, and since it's just a key we wouldn't describe it as detailed.

Reply:
Aye, remnant. Same with the as_json below.